pages/management/ManagementHandler.inc.php

```diff
@@ -273,6 +276,7 @@ public function workflow($args, $request)
             'components' => [
                 FORM_AUTHOR_GUIDELINES => $authorGuidelinesForm->getConfig(),
                 FORM_METADATA_SETTINGS => $metadataSettingsForm->getConfig(),
+                PKPDoiSettingsForm::FORM_DOI_SETTINGS => $doiSettingsForm->getConfig(),
```

Comment:
You can use `DoiSettingsForm::FORM_DOI_SETTINGS` here and get rid of the `use` statement at the top.